src/pkg/services/m365/api/mail_test.go

```diff
@@ -411,3 +397,32 @@ func (suite *MailAPIIntgSuite) TestRestoreLargeAttachment() {
 	require.NoError(t, err, clues.ToCore(err))
 	require.NotEmpty(t, id, "empty id for large attachment")
 }
+
+func (suite *MailAPIIntgSuite) TestMail_GetContainerByName() {
+	table := []struct {
+		name      string
+		expectErr assert.ErrorAssertionFunc
+	}{
+		{
+			name:      "Inbox",
```

Comment:
Can we add a test to check for type sensitivity?

Reply:
These are case insensitive.